site/content/docs/developing/internal-traffic.en.md

```diff
@@ -1,12 +1,14 @@
-# Service Discovery
+# Internal Traffic
+
+## Service Discovery
 
```

Comment:
and, iirc, Service Connect is free whereas Service Discovery isn't, right?

Reply:
> Can we add an [info admonition here](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#supported-types) recommending folks that they should prefer using service connect which was added with Copilot v1.24.

But all the info's been mentioned in the doc 🤔 

> and, iirc, Service Connect is free whereas Service Discovery isn't, right?

I think both of them are "free", and both of them are subject to additional cost (sc could be even higher because of the additional cost caused by the envoy sidecar usage)